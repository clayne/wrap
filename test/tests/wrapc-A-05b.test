wrapc | /dev/null | -A41 -D/* | wrapc-A-05b.c | 0
