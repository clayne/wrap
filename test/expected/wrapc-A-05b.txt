static size_t       a_very_long_variable_name;/* max width of line */
